ci/rt.sh

```diff
@@ -59,21 +68,35 @@ run_hrrr=yes
 run_hafs=yes
 run_rtma=yes
 
+# Tests with IFI enabled only work if libIFI is present.
+if [[ "$have_ifi" == yes && "$disable_ifi" == no ]] ; then
+  run_hrrr_ifi=yes
+  run_ifi_standalone_hrrr=no # turned off by default because it doesn't match UPP output yet
+  run_fv3r_ifi=yes
+  run_ifi_standalone_fv3r=yes
+else
+  # Cannot run these without ifi
+  run_hrrr_ifi=no
+  run_ifi_standalone_hrrr=no
+  run_fv3r_ifi=no
+  run_ifi_standalone_fv3r=no
+fi
+
 #find machine
 mac=$(hostname | cut -c1-1)
 mac2=$(hostname | cut -c1-2)
 mac3=$(hostname | cut -c1-4)
 if [ $mac2 = hf ]; then # for HERA
  export machine=HERA
- export homedir=${homedir:-"/scratch2/NAGAPE/epic/UPP/test_suite"}
+ export homedir=${homedir:-"/scratch2/BMC/ifi/Samuel.Trahan/test_suite/"}
```

Comment:
@SamuelTrahanNOAA Will this type of path be changed to one from EPIC side after we complete testing?

Reply:
It must be reverted before the pull request is merged.

Reply:
After the files are copied to epic areas, we should re-run the regression tests to ensure the files are all there correctly.

Reply:
@gspetro-NOAA Could you sync Sam's changes from his 'test_suite' to EPIC's 'test_suite' so we could rerun tests?